chromadb/migrations/metadb/00003-embedding-metadata.sqlite.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE embedding_metadata_lists (
+    id INTEGER REFERENCES embeddings(id),
+    key TEXT NOT NULL,
+    string_value TEXT,
+    float_value REAL,
+    int_value INTEGER,
+    bool_value BOOLEAN,
```

Comment:
would be odd to have a list of bools, but the implementation is simpler if this schema matches `embedding_metadata`